tools/sapp/sapp/interactive.py

```diff
@@ -445,6 +445,10 @@ def issues(
         codes: Optional[Union[int, List[int]]] = None,
         callables: Optional[Union[str, List[str]]] = None,
         filenames: Optional[Union[str, List[str]]] = None,
+        minTraceLength_to_sources: Optional[Union[int, List[int]]] = None,
```

Comment:
I'm going to be an awful person, and suggest that we use a less precise name :) How would you feel about renaming `minTraceLength -> traceLength`? For the simple case (one path from source/to sink), it coincides with people's intuition, and the `minTraceLength` isn't an obvious concept unless you know how these tools work - so I think the term warrants a comment in docs either way.

The second suggestion I'd make is renaming these to be `exact` and `max` instead of `<empty string>` and `max`. I don't think it's currently clear that the `minTraceLength_to_sources` is an exact match, so might confuse the n00b :)

All in all, here's what the suggested names look like:

```
max_trace_length_to_sinks
exact_trace_length_to_sinks
max_trace_length_to_sources
exact_trace_length_to_sources
```

I'm not attached to these names at all, and you might have better ideas on naming these (after all, you're working with these daily :)) but hopefully the concern makes sense!

Reply:
Yes, that all make sense. Totally agreed that the `max` in the min trace length seems a bit confusing :). I think those suggested names works for us. I'll update the code to match those parameters.